src/api/app/components/bs_request_comment_component.html.haml

```diff
@@ -21,8 +21,17 @@
             = link_to('#', data: { 'bs-toggle': 'modal', 'bs-target': "#delete-comment-modal-#{comment.id}" },
                       class: 'dropdown-item delete_link', title: 'Delete comment') do
               Delete
-
-    = helpers.render_as_markdown(comment)
+    - if comment.diff_ref && diff
+      - action = comment.commentable
+      .px-3.py-2
+        = link_to request_show_path(number: action.bs_request, request_action_id: action) do
+          = action.uniq_key
+        >
+        = link_to request_show_path(number: action.bs_request, request_action_id: action, anchor: comment.diff_ref) do
+          = render(DiffSubjectComponent.new(state: diff['state'], old_filename: diff.dig('old', 'name'), new_filename: diff.dig('new', 'name')))
+      = render(DiffComponent.new(diff: diff.dig('diff', '_content'), range:))
```

Comment:
There is also a weird behaviour/look after we edit this type of comment or Reply to it.

Please look at the GIF:

![Peek 2023-03-27 12-24](https://user-images.githubusercontent.com/2581944/227925731-e6914810-74c0-4875-86ce-36f9e0aef6ed.gif)

Reply:
In isolation, I also like the darker border, but that will bring back many dark lines in the view, which we managed to avoid in the last iteration.

Rough draft of the result:

![Screenshot 2023-03-27 at 16-18-47 Open Build Service](https://user-images.githubusercontent.com/2581944/227967477-10f0405e-9878-4a2f-b411-2822d4616ba2.png)

Reply:
Ok, let's leave it like that :+1: